axiom/nr_segment_datastore.h

```diff
@@ -22,6 +22,7 @@ typedef struct _nr_segment_datastore_params_t {
                        extracted from the SQL for SQL segments. */
   char* operation;  /* The null-terminated operation; if NULL, this will be
                        extracted from the SQL for SQL segments. */
+  char* db_system;  /* Database management system (DBMS) product being used.*/
```

Comment:
Is this only currently populated for DynamobDB?
Do we need to add support for existing DBMS as well in another effort?

Reply:
Yes this is only currently populated for DynamoDB to provide minimal impact to customers since it provides duplicate information to datastore.component attribute but was required to build the relationship.